Docker/README.md

```diff
@@ -626,15 +661,17 @@ httpd: Could not reliably determine the server's fully qualified domain name, us
 ```
 1. What happened ? Why don't you get a prompt ?
 2. Use `docker ps` to see the status of your container and `docker logs` to see what happened.
-3. Try to adapt the dockerfile to solve that issue. **Discuss with your trainer if you're stuck !**
+3. Try to adapt the Dockerfile to solve that issue. **Discuss with your trainer if you're stuck !**
 
 `#` **`perl -pi -e 's|D httpd|D /usr/sbin/apachectl -DFOREGROUND -k start|' Dockerfile`**
-(This magic command replaces the launch of the httpd command by the apachectl one with the right options. In case you use RHEL 7, you need to install perl with yum)
+(This magic command replaces the launch of the httpd command by the apachectl one with the right options. If you are using RHEL 7 as the host OS, you will have to install perl via yum.)
 
 1. Try to use a browser (you may want to install lynx) to connect to your web server. Can you do it ?
-2. Which IP address do you point to ? You may use `docker exec` to see that IP address.
+2. Which IP address do you point to ? You can use `docker exec` to connect to the container and get the IP address for the container.
```

Comment:
You don't really "connect" to a container. You place yourself in the execution environment of the container.